### ISSUE TITLE
Painting and Pan functionality conflicting with left-click

### ISSUE THREAD

**Issue by pdeperio** [WatChPhoto/pmt-annotation#25]
This might just be an issue with my GUI, but if I want to paint by clicking and holding left click, this actually pans as well making it very difficult to paint. @martindgr do you not have this problem?

**Comment by msekatchev** [comment]
Are you still able to pan while holding down shift?
We could implement something that disables painting while the shift key is pressed, allowing you to pan with shift+left-click and still paint with left-click/middle-click

**Comment by pdeperio** [comment]
Yes I can still pan, that's not the problem. Problem is I want to paint, but I can't because it's panning. (I struggled a lot trying to figure this out in the guts of the `qt` code, so can be lower priority if @martindgr is alright.)

**Comment by martindgr** [comment]
Ah the panning issue hadn't occurred to me, having just had a play around with it, pressing left and right mouse at the same time allows me to paint without panning. Though right mouse used to bring up a menu so im not sure if it is a solid fix but it works for now so Im happy having it as low priority
